mxd/postman/management_api_tests.json

```diff
@@ -386,83 +533,150 @@
 			],
 			"request": {
 				"method": "POST",
-				"header": [
-					{
-						"key": "X-Api-Key",
-						"value": "password",
-						"type": "text"
-					}
-				],
+				"header": [],
 				"body": {
 					"mode": "raw",
-					"raw": "{\r\n    \"@context\": {\r\n        \"edc\": \"https://w3id.org/edc/v0.0.1/ns/\"\r\n    },\r\n    \"@type\": \"CatalogRequest\",\r\n    \"providerUrl\": \"\",\r\n    \"counterPartyAddress\":\"{{Alice_PROVIDER_PROTOCOL_URL}}\",\r\n    \"protocol\": \"dataspace-protocol-http\",\r\n    \"querySpec\": {\r\n        \"offset\": 0,\r\n        \"limit\": 50\r\n    }\r\n}",
+					"raw": "{\n  \"@context\": {\n    \"odrl\": \"http://www.w3.org/ns/odrl/2/\"\n  },\n  \"assetId\": \"{{ASSET_ID}}\",\n  \"connectorAddress\": \"{{PROVIDER_PROTOCOL_URL}}\",\n  \"connectorId\": \"{{PROVIDER_ID}}\",\n  \"contractId\": \"{{CONTRACT_AGREEMENT_ID}}\",\n  \"dataDestination\": {\n    \"type\": \"HttpProxy\"\n  },\n  \"privateProperties\": {\n    \"receiverHttpEndpoint\": \"{{BACKEND_SERVICE_PROTOCOL}}/api/v1/transfers\"\n  },\n  \"protocol\": \"dataspace-protocol-http\"\n}",
 					"options": {
 						"raw": {
 							"language": "json"
 						}
 					}
 				},
 				"url": {
-					"raw": "{{BOB_MANAGEMENT_URL}}/v2/catalog/request",
+					"raw": "{{CONSUMER_MANAGEMENT_URL}}/v2/transferprocesses",
 					"host": [
-						"{{BOB_MANAGEMENT_URL}}"
+						"{{CONSUMER_MANAGEMENT_URL}}"
 					],
 					"path": [
 						"v2",
-						"catalog",
-						"request"
+						"transferprocesses"
 					]
 				}
 			},
 			"response": []
 		},
 		{
-			"name": "Query Catalog Alice to BOB",
+			"name": "Get Transfer by ID",
 			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							"const API_KEY_HEADER = 'X-Api-Key';\r",
+							"const URL=`${pm.collectionVariables.get(\"CONSUMER_MANAGEMENT_URL\")}/v2/transferprocesses/${pm.collectionVariables.get(\"TRANSFER_ID\")}/state`;\r",
+							"function delay(ms) {\r",
+							"  return new Promise(resolve => setTimeout(resolve, ms));\r",
+							"}\r",
+							"\r",
+							"async function makeRequests() {\r",
+							"  for (let i = 0; i < 10; i++) {\r",
+							"    const response = await new Promise(resolve => {\r",
+							"      pm.sendRequest({\r",
+							"        url: URL,\r",
+							"        method: 'GET',\r",
+							"        header: {\r",
+							"          ...pm.request.headers.toObject(),\r",
+							"          [API_KEY_HEADER]: pm.collectionVariables.get(\"API_KEY_PASSWORD\"),\r",
+							"        },\r",
+							"      }, (error, response) => {\r",
+							"        resolve(response);\r",
+							"      });\r",
+							"    });\r",
+							"\r",
+							"    var data = response.json();\r",
+							"    if (data[\"edc:state\"] === \"STARTED\") {\r",
```

Comment:
Field name is `state` instead of `edc:state`

Reply:
I have updated from **edc:state** to **state** in all the required places.